plugin-test/build.gradle

```diff
@@ -0,0 +1,53 @@
+/*
+    The MIT License (MIT)
+    
+    ${PROJECT_NAME} Copyright (c) ${YEAR} ${USER}
```

Comment:
I don't want to blame anyone, but i think that the license headers are *bad* thing. Here're some arguments against writing them:

1. You need to update the copyright year from time to time (otherwise it'll be pointless). And updating the copyright year in every file is just a huge pointless commit you need to make every year.
2. You need to update the author list in every single file. These're tricky and tend to diverge momentary. Boom — and you already have hundred of different headers with different (possibly mistyped or lost in translation or corroded by encoding conversion) author names. That's chaos to support and sort out.
3. Sometimes, when changing something, you forget to update the headers. Boom — and you've left some GPL-licensed files in a MIT project. Yes, that's a legal problem. Yes, that's a huge thing (especially when the future maintainers won't be able to contact you).

I had all sorts of problems with the license headers within my projects (or in the orphaned projects adopted by me), that's why I care so much.

Overall: the less maintainership work the license will require — the better. So, please, remove them if you don't mind.